panda.config.ts

```diff
@@ -17,17 +23,15 @@ const pandaConfig = defineConfig({
   studio: {
     logo: "https://github.com/animareflection/ui/blob/master/public/img/logo.png?raw=true",
   },
-  // TODO remove static CSS pregeneration
-  staticCss: {
-    recipes: {
-      accordion: [{ variant: ["*"] }],
-      alert: [{ variant: ["*"] }],
-      hide: [{ variant: ["*"] }],
-      show: [{ variant: ["*"] }],
-      stat: [{ variant: ["*"] }],
-      tooltip: [{ variant: ["*"] }],
```

Comment:
Notice that these were applied to the library bundle itself, and now every static generation is blocked behind the env var. Make sure to test these components out in the example apps locally to make sure this did not cause a regression